frontend/sass/main.sass

```diff
@@ -136,4 +136,7 @@ md-datepicker.ng-pristine .md-datepicker-invalid
     width: 640px
 
 md-list-item.selected
-  background-color: rgba(170, 170, 170, 0.1)
\ No newline at end of file
+  background-color: rgba(170, 170, 170, 0.1)
+
+s
```

Comment:
Oh, no this is the <s> element, which is a strikethrough showing no longer correct data.